src/Controls/tests/UITests/Tests/Issues/Issue18726.cs

```diff
@@ -0,0 +1,46 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue18726 : _IssuesUITest
+	{
+		public Issue18726(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "Editor custom Keyboard works";
+
+		[Test]
+		public void EditorCustomKeyboardWorks()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.iOS, TestDevice.Mac, TestDevice.Windows },
```

Comment:
The Keyboard property is not working correctly on any platform. 

![image](https://github.com/dotnet/maui/assets/6755973/ec5afb2c-5997-4fc0-a138-bb75b22111d3)
![image](https://github.com/dotnet/maui/assets/6755973/772e0859-3a83-460e-a3fc-a92639eaf55c)

Reply:
Should we close this PR and log an issue?

Reply:
We already have https://github.com/dotnet/maui/issues/17784
And this ignored tests could be useful when someone starts working on that issue.